airflow/api_internal/endpoints/rpc_api_endpoint.py

```diff
@@ -51,7 +51,7 @@ def _initialize_map() -> dict[str, Callable]:
         Variable.update,
         Variable.delete,
     ]
-    return {f"{func.__module__}.{func.__name__}": func for func in functions}
+    return {f"{func.__module__}.{func.__qualname__}": func for func in functions}
```

Comment:
```suggestion
    return {qualname(func): func for func in functions}
```
Then we can import `qualname` from airflow/utils/module_loading

Reply:
Unfortunatelly it doesn't work.
for functions it uses the old format (`__name__` instead of  `__qualname__`)
https://github.com/apache/airflow/blob/main/airflow/utils/module_loading.py#L47